aqua/qt6/Portfile

```diff
@@ -822,6 +815,10 @@ foreach {module module_info} [array get modules] {
                     -DCMAKE_OBJC_COMPILER_LAUNCHER=${prefix}/bin/ccache \
                     -DCMAKE_OBJCXX_COMPILER_LAUNCHER=${prefix}/bin/ccache
             }
+            if {[variant_isset universal]} {
+                configure.post_args-append \
+                    "\"-DCMAKE_OSX_ARCHITECTURES=x86_64;arm64\""
```

Comment:
I guess this is correct as Qt6 only works on macOS 10.14 and up so there are probably no other "universal" combination (@kencu would you mind looking at this PR?)

Reply:
Yeah, it's the only one supported by Qt6.  I could go for the `[join ${configure.universal_archs} ;]` I've seen on other portfiles, but do you know what the equivalent is for non-universal builds since I'd need that as well (or else we'd still be effectively hardcoding the support to x86_64 and arm64 in the `else` of this if statement, defeating the purpose of being generic).